ctci/chapter1/1.9.py

```diff
@@ -0,0 +1,20 @@
+#1.9
+#String Rotation: Assume you have amethod isSubstring which checks 
+#ifone word is asubstring of another. Given two strings, s1 and s2, 
+#write code to check if s2 is a rotation of s1 using only one call to 
+#isSubstring (e.g.,"waterbottle"is a rotation of"erbottlewat").
+
+#in keyword is bascially substring
+def isUnisSubstring(s1: str, s2: str) -> bool:
```

Comment:
Is there a typo in the method name?

Reply:
I believe this should be isRotation() is what he meant?